kuzzleio/kuzzle-device-manager
# Feat/assets groups last update

## What does this PR do ?

Improve assets groups to add the data `lastUpdate` to store when groups have been updated

### How should this be manually tested?

  - Step 1 : update of a group from API
  - Step 2 : check `lastUpdate` property has been update
